### PR TITLE
Remove Nitro module

### PR DESCRIPTION
**Pull request by xjunior**

### CODE REVIEW

**Comment by wadewinningham** [comment]
@xjunior There a reason for this? I was going to create a `Nitro::PivotalTracker` class. There a reason it shouldn't be a `Nitro` module?

**Comment by xjunior** [comment]
@wadewinningham this is going open-source. It has no connections to Nitro, but is a gem used by nitro.

File: lib/consent/dsl.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    def initialize(subject, defaults)
+      @subject = subject
+      @defaults = defaults
+    end
+
+    def with_defaults(new_defaults, &block)
+      DSL.build(@subject, @defaults.merge(new_defaults), &block)
+    end
+
+    # rubocop:disable Link/UnusedBlockArgument, Link/Eval
```
Comment (gmfvpereira) [inline lib/consent/dsl.rb]: Do you have rubocop?

**Comment by xjunior** [comment]
Good catch, @gmfvpereira . Removed those lines on https://github.com/powerhome/consent/pull/4/commits/1bbcba2ceb565a5cc705ed6930243cd8cbf961bf.

File: lib/consent/permissions.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    def initialize(permissions)
+      @permissions = permissions
+    end
+
+    def each(&block)
+      Consent.subjects.values.each do |subject|
+        subject.actions.map do |action|
+          map_permission subject, action
+        end.compact.each(&block)
```
Comment (jilltank) [inline lib/consent/permissions.rb]: would it be possible to write this as `subject.actions.map { |action| map_permission(subject, action) }.compact.each(&block)`  ?

I know it's a slightly long line, but this way you avoid the extra nesting here

**Pull request merged**